src/main/java/org/jabref/logic/journals/JournalAbbreviationRepository.java

```diff
@@ -48,7 +49,8 @@ private static boolean isMatchedAbbreviated(String name, Abbreviation abbreviati
      * Letters) or its abbreviated form (e.g. Phys. Rev. Lett.).
      */
     public boolean isKnownName(String journalName) {
-        String journal = journalName.trim();
+        // Trims the String and also replaces any instances of "\&" with "&" for abbreviation search purposes
```

Comment:
`Trims the String and also replaces any instances of "\&" with "&"` is a literal explanation of the code and can be removed.

```suggestion
        // The journal lists contains unescaped  only, thus "\&" is replaced by "&" for abbreviation search purposes
```

Reply:
Removed!